Resources/Prototypes/WhiteDream/Entities/Objects/Cult/constructs.yml

```diff
@@ -107,6 +107,8 @@
     understands:
     - TauCetiBasic
     - Eldritch
+  - type: LanguageSpeaker
```

Comment:
Don't forget to add `LanguageSpeaker` for soul shards too in `Resources/Prototypes/WhiteDream/Entities/Objects/Cult/souls_shards.yml`

Reply:
thanks